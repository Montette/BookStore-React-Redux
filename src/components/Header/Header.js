import React from 'react';
import './Header.scss';

const Header = (props) => {
    
    return (
        <header className='header'>
            <h1 className='header__title'>Book Store</h1>
        
        </header>
    );
}

export default Header;