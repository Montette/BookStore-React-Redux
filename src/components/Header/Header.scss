.header {
    height: 70px;
    background-color: #24292e;
    color: #fff;
    position: absolute;
    top: 0;
    left: 0;
    width: 100%;
    display: flex;
    justify-content: space-between;
    align-items: center;
    padding: 0 30px;
    
    &__title {
        margin: 0;
        line-height: 70px;
    }
}






