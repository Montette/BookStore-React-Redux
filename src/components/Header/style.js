import styled from 'styled-components';

export const StyledHeader = styled.header `
    height: 70px;
    background-color: #24292e;
    color: #fff;
`
export const Title = styled.h1 `
    margin: 0;
    line-height: 70px;
`