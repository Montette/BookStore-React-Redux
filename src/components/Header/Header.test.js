import React from 'react';
import ReactDOM from 'react-dom';
import Header from './Header';

import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure ({adapter: new Adapter()})

describe('header tests', ()=> {


    it('header renders without crashing', ()=> {
        shallow(<Header />)
    })



})