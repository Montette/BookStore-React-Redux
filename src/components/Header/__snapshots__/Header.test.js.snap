// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header tests Snapshot matches 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h1
        className="header__title"
>
        Book Store
</h1>,
      "className": "header",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Book Store",
        "className": "header__title",
      },
      "ref": null,
      "rendered": "Book Store",
      "type": "h1",
    },
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h1
          className="header__title"
>
          Book Store
</h1>,
        "className": "header",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Book Store",
          "className": "header__title",
        },
        "ref": null,
        "rendered": "Book Store",
        "type": "h1",
      },
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
